examples/management_canister/src/index.ts

```diff
@@ -113,6 +115,21 @@ export default Canister({
 
         return true;
     }),
+    getCanisterInfo: update(
+        [CanisterInfoArgs],
+        CanisterInfoResult,
+        async (args) => {
```

Comment:
Just pass the args into the cross canister call

Reply:
Turns out I greatly exaggerated, and canisterStatus was really the only other place this happened...

Both are fixed.